### PR TITLE
Minor fix to deadchat discord ooc

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
Minor fix to deadchat discord ooc. caused by https://github.com/BeeStation/BeeStation-Hornet/pull/7940

## Why It's Good For The Game
minor bug fix

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

it tells the message length is now 4

</details>

## Changelog
:cl:
fix: discord deadchat message is now properly formatted
/:cl:

### CODE REVIEW

File: code/modules/client/verbs/ooc.dm [hunk 1]
```diff
@@ -112,6 +112,7 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 			discordsendmsg("ooc", "[discord_ooc_tag](OOC) **[sayer]:** [msg]")
 		if(CHAT_TYPE_DEADCHAT) // don't send these until a round is finished
 			if(SSticker.current_state == GAME_STATE_FINISHED)
+				msg = copytext(msg, 24, length(msg)-7) // wipes some bad dchat format
```
Comment (itsmeow) [inline code/modules/client/verbs/ooc.dm]: this seems really ugly and bound to break, it should use regex to parse out what it wants instead

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/modules/client/verbs/ooc.dm (hunk 1 above)
Comment (EvilDragonfiend) [reply code/modules/client/verbs/ooc.dm]: simple is best. If it is broken in the future, a regex will be broken as well, as a regex for this isn't very nice at all.

Comment (EvilDragonfiend) [reply code/modules/client/verbs/ooc.dm]: aight, gonna change it to regex

**Comment by EvilDragonfiend** [comment]
regex'ed

File: code/modules/client/verbs/ooc.dm [hunk 2]
```diff
@@ -112,6 +112,10 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 			discordsendmsg("ooc", "[discord_ooc_tag](OOC) **[sayer]:** [msg]")
 		if(CHAT_TYPE_DEADCHAT) // don't send these until a round is finished
 			if(SSticker.current_state == GAME_STATE_FINISHED)
+				var/regex/R = regex("<span class=' '>(.+)</span>\"")
```
Comment (itsmeow) [inline code/modules/client/verbs/ooc.dm]: `.` will not work for this, it needs to be [\s\S] to match any character including whitespace

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Review by itsmeow — APPROVED** [review]

**Pull request merged**